### DISCUSSION TITLE
What is a "column"

### DISCUSSION THREAD

**Discussion by andy-maier** [mikefarah/yq#1958] [Q&A]
Can someone please explain what the concept of a "column" is?

The description at https://mikefarah.gitbook.io/yq/operators/column does not explain it at all, and I cannot make any sense of the examples.

**Comment by mikefarah** [comment]
Sorry I should clarify that documentation. Column refers to the character position in the document (e.g. line 3 means it's on the 3 line of the document, counting the 'rows'; column 3 means it 3 spaces across (so there are 3 characters preceding it on that line).
